pytest/tests/sanity/restaked.py

```diff
@@ -53,12 +56,16 @@ def start_restaked(node_dir, rpc_port, config):
 
 started = time.time()
 
+# Wait until at least one block is generated since otherwise we will get
+# UnknownEpoch errors inside of restaked binary when asking for validators info.
+while not (b := get_height(nodes[1])):
```

Comment:
I think we have a library util for that:
wait_for_blocks_or_timeout

Reply:
Yes and no. `wait_for_blocks_or_timeout` waits for given number of new blocks to appear. So if there were already 5 blocks produced and we call it to wait for one block it will wait for 6th block. However, here we just want for there to be at least on block and if there’s one ready that works for us.

Anyway, could we get this merged while I’m preparing a larger refactor for wait for blocks and some other code.